sonar-flex-plugin/src/test/java/org/sonar/plugins/flex/cobertura/CoberturaSensorTest.java

```diff
@@ -55,14 +56,7 @@ public void setUp() throws Exception {
 
   @Test
   public void shouldParseReport() throws Exception {
-    String content = new String(Files.readAllBytes(Paths.get(TEST_DIR, "src/example/File.as")), StandardCharsets.UTF_8);
```

Comment:
I don't see how is this change related to the PR

Reply:
I extracted the code in a separate method to reuse it in a new unit test.